lib/authority/controller.rb

```diff
@@ -144,6 +148,15 @@ def authority_forbidden(error)
       render :file => Rails.root.join('public', '403.html'), :status => 403, :layout => false
     end
 
+    # Log successful authorization attempt if enabled in the authority configuration.
+    #
+    # This method can be overloaded inside the application controller, similar to authority_forbidden.
+    def authority_success(user, action, resource)
+      if Authority.configuration.log_success
+        Authority.logger.info("#{user} successfully performed the #{action} action to this resource: #{resource}")
```

Comment:
1. Fair point. I think the function name makes sense, since we successfully passed through authority.. but the default message should be changed. Will fix that.

2. For my personal need I have to overwrite it because I need to capture specific information in the request such as path details and controller methods. Also figured that if it were a controller method other users would see the need to extend it as well since it's similar to the `authority_forbidden` action. At first I did have this being logged via `Authority.enforce`, but overriding that was less clean than just making it a controller method.

Really up to you! It's easy to make the change, but I think it's more restrictive.